patroni/postgresql.py

```diff
@@ -1367,16 +1369,16 @@ def save_configuration_files(self):
         try:
             for f in self._configuration_to_save:
                 if os.path.isfile(f):
-                    shutil.copy(f, f + '.backup')
+                    shutil.copy(f, f.replace(self._config_dir, self._data_dir) + '.backup')
```

Comment:
What if `hba_file` and/or `ident_file` are in the `custom_conf` file? Are we supposed to override those settings by writing them after the `include` or before as a "default" value?